lib/tasks/gather.js

```diff
@@ -1090,7 +1094,7 @@ class TaskGather extends SttTask {
         }
       }
 
-      if (!emptyTranscript) {
+      if (!emptyTranscript && evt.alternatives[0].transcript.trim() !== '') {
```

Comment:
if we ignore like this.
Can you please set emptyTranscript = evt.alternatives[0].transcript.trim() === '' as well.
we cannot have emptyTranscript = true, but evt.alternatives[0].transcript.trim() === '', this will break system.
let make consisten here.